crates/bitwarden-uniffi/src/platform/repository.rs

```diff
@@ -64,12 +63,7 @@ macro_rules! create_uniffi_repository {
             async fn has(
                 &self,
                 id: String,
-            ) -> Result<bool, $crate::platform::repository::RepositoryError> {
-                match self.get(id).await {
-                    Ok(x) => Ok(x.is_some()),
-                    Err(e) => Err(e),
-                }
-            }
+            ) -> Result<bool, $crate::platform::repository::RepositoryError>;
```

Comment:
The latest UniFFI doesn't allow default trait methods, so I've removed it. We weren't using it anyway.